src/main/groovy/wooga/gradle/unity/UnityPlugin.groovy

```diff
@@ -403,9 +417,53 @@ class UnityPlugin implements Plugin<Project> {
     }
 
     private static void configurePackageTasks(UnityPluginExtension extension, final Project project) {
-        // TODO: Perhaps hook the task to generate meta files
-        project.tasks.withType(GenerateUpmPackage).configureEach({ t ->
-        })
+
+        // Only execute for >= 2018.3
+        def upmIsSupported = new Spec<Task>() {
+            @Override
+            boolean isSatisfiedBy(Task o) {
+                def _unityVersion = getUnityVersion(extension, project)
+                Boolean supported = _unityVersion.majorVersion > 2018 ||
+                    (_unityVersion.majorVersion == 2018 && _unityVersion.minorVersion > 3)
+                if (!supported) {
+                    project.logger.warn("The unity version (${_unityVersion.majorVersion}.${_unityVersion.minorVersion}) " +
+                        "does not support UPM packages, skipping")
+                }
+                supported
+            }
```

Comment:
Can we please not start writing variables with leading `_` please.

Reply:
Sure.